packages/react/src/experimental/Skeleton/SkeletonBox.tsx

```diff
@@ -36,5 +39,15 @@ export const SkeletonBox = styled.div<SkeletonBoxProps>`
     outline-offset: -1px;
   }
 
+  ${space};
+  ${color};
+  ${typography};
+  ${layout};
+  ${flexbox};
+  ${grid};
+  ${background};
+  ${border};
+  ${position};
+  ${shadow};
```

Comment:
We don't prefer system props in the API anymore. We recommend using sx or css modules

```suggestion
-  ${space};
-  ${color};
-  ${typography};
-  ${layout};
-  ${flexbox};
-  ${grid};
-  ${background};
-  ${border};
-  ${position};
-  ${shadow};
```